content/docs/guide-development.md

```diff
@@ -1,36 +1,36 @@
-# Workflow development Guide
+# [MMODA](https://www.astro.unige.ch/mmoda) workflow development Guide
 
-Workflows are all things that can be computed, broadly speaking. 
+The [MMODA](https://www.astro.unige.ch/mmoda) platform provides access to the Astronomical Open Research Data Analysis Services (ORDAS). Good fraction of these services follow a simple scheme: they  
+* access publicly available external astronomical data archives to fetch data relevant to specific source or source catalog,
+* transform the fetched data using a workflow based on a Python notebook, to drive a data product and
+The users of [MMODA](https://www.astro.unige.ch/mmoda) are encouraged to become its developers and add new ORDAS. This help page provides a step-by step instructions on how to add new services to MMODA.
+* display a preview of the data product on the [MMODA](https://www.astro.unige.ch/mmoda) frontend and/or return the data product to the user via  Python [API](https://github.com/oda-hub/oda_api)
 
-For reproducibility, we want our workflows to be repeatable: producing the same output every time they are computed. 
-This is easy enough to do in first approximation, but might be harder to achieve than it seems when the workflow relies on external resources. But we track every execution so it is not necessary to be overly concerned about these delicate details at every moment.
+The users of [MMODA](https://www.astro.unige.ch/mmoda) are encouraged to become its developers and add new ORDAS. This help page provides a step-by step instructions on how to add new services to [MMODA](https://www.astro.unige.ch/mmoda) .
 
-Generally, we want the workflows to be parametrized. Non-parametrized but strickly repeatable notebooks are less **reusable** since they always produce the same output data.
+Workflows are all things that can be computed, broadly speaking. For reproducibility, we want our workflows to be repeatable: producing the same output every time they are computed. This looks easy enough in first approximation, but might be harder to achieve than it seems when the workflow relies on external data and compute resources  and has to yield an "up-and-running" ORDAS in an ever-evolving compute environment. This help page is also aimed at helping the developers in ensuring reproducibility and reusability of their workflows converted to ORDAS.
```

Comment:
this misses a bit the point which was there, about that to reproduce it is not enough to just repeat, to properly reproduce (and hence reuse with trust) it is almost always necessary to change something.